docs/build-node-interaction.md

```diff
@@ -0,0 +1,163 @@
+  {'type': 'Seal',
+   'index': 5,
+   'value': [1161969986,
+    '0x14645b478c91e081e0d0fb23c69bf19b367c418f95111a92edbd6791ab0a9907bde61d2330f97100bf9e2316f7a21d4d62d37878c54ae6d3c870a2844ada5087']}],
+ 'extrinsics': [{'method': 'timestamp.set',
+   'signature': None,
+   'nonce': 0,
+   'args': [1584104340000],
+   'tip': 0,
+   'hash': '0xb4a40086f7203b82a69e4ff10c839c2e23ade50c8d0d1ab7521a619d773e426e',
+   'info': {},
+   'events': [{'method': 'system.ExtrinsicSuccess',
+     'data': [{'weight': 10000, 'class': 'Operational', 'paysFee': True}]}],
+   'success': True},
+  {'method': 'finalityTracker.finalHint',
+   'signature': None,
+   'nonce': 0,
+   'args': [1432508],
+   'tip': 0,
+   'hash': '0x4d3925dbb29f5938de2682d11809feb708051bc748141e70e713fe9db7950c95',
+   'info': {},
+   'events': [{'method': 'system.ExtrinsicSuccess',
+     'data': [{'weight': 10000, 'class': 'Normal', 'paysFee': True}]}],
+   'success': True},
+  {'method': 'parachains.setHeads',
+   'signature': None,
+   'nonce': 0,
+   'args': [[]],
+   'tip': 0,
+   'hash': '0xcf52705d1ade64fc0b05859ac28358c0770a217dd76b75e586ae848c56ae810d',
+   'info': {},
+   'events': [{'method': 'system.ExtrinsicSuccess',
+     'data': [{'weight': 1000000, 'class': 'Normal', 'paysFee': True}]}],
+   'success': True},
+  {'method': 'balances.transfer',
+   'signature': {'signature': '0xd6d0ec4828bbe08efce3f400ef61b5fc9b4c7c6381776fb2a4dfd2b8b40193857161d2ac0ac451357f4682fb3ddff38d6850fe9a282a25119f20ad4dc6ba0b0f',
+    'signer': 'Fy2KSMaRjkpxpJ7UFhCiWqXrFuKdRfurtEn24hYJNr5oJLb'},
+   'nonce': 0,
+   'args': ['G9njmrCmZKBnojVue8umYiVLGF8W7eJJWTkUZwWktmQxEFo', 90000000000],
+   'tip': 0,
+   'hash': '0x7cb3ac653e0c17131bf4d952dbe0db28f88faa1e112b193236684d67d76dc4c1',
+   'info': {'weight': 1000000, 'class': 'Normal', 'partialFee': 10000000000},
+   'events': [{'method': 'treasury.Deposit', 'data': [8000000000]},
+    {'method': 'balances.Deposit',
+     'data': ['FXCgfz7AzQA1fNaUqubSgXxGh77sjWVVkypgueWLmAcwv79', 2000000000]},
+    {'method': 'system.KilledAccount',
+     'data': ['Fy2KSMaRjkpxpJ7UFhCiWqXrFuKdRfurtEn24hYJNr5oJLb']},
+    {'method': 'balances.Transfer',
+     'data': ['Fy2KSMaRjkpxpJ7UFhCiWqXrFuKdRfurtEn24hYJNr5oJLb',
+      'G9njmrCmZKBnojVue8umYiVLGF8W7eJJWTkUZwWktmQxEFo',
+      90000000000]},
+    {'method': 'system.ExtrinsicSuccess',
+     'data': [{'weight': 1000000, 'class': 'Normal', 'paysFee': True}]}],
+   'success': True}]}
+```
+
+### Submitting a Transaction
+
+Submit a serialized transaction using the `tx` endpoint. This endpoint returns a JSON with the
+transaction hash.
+
+```python
+import requests
+import json
+
+url = 'http://127.0.0.1:8080/tx/'
+response = requests.post(url, data='{"tx": "0x..."}')
+if response.ok:
+	txhash = json.loads(response.text)
```

Comment:
the response body is also going to be a json, with tx hash being under the key `"hash"`:

```
{
    "hash": "..." 
}
```

In case there is an error (either user input or rejected by the node) you will get an error report instead:

```
{
    "error": "Failed to parse a tx" | "Failed to submit a tx",
    "cause": "Upstream error description"
}
```